primeiros-passos-com-go/ponteiros-e-erros/ponteiros-e-erros.md

```diff
@@ -521,7 +521,7 @@ func TestCarteira(t *testing.T) {
 		erro := carteira.Retirar(Bitcoin(10))
 
 		confirmaSaldo(t, carteira, Bitcoin(10))
-		confirmaErroInexistente(t, erro)
+		confirmaErro(t, erro, ErroSaldoInsuficiente)
```

Comment:
Neste teste nenhum erro é esperado

Reply:
So just remove the function ?  because the function "confirmaErroInexistente" doesn't exist in this time of the tutorial. It is declared after the next topic.

Reply:
Good point, didn't notice that.
Looking at the original version, the error was not checked here, before defining the helper function, so I think we should remove this.

Reply:
Ok, I'll remove it